src/ethereum_clis/clis/execution_specs.py

```diff
@@ -179,6 +179,14 @@ def _mapping_data(self):
                 TransactionException.PRIORITY_GREATER_THAN_MAX_FEE_PER_GAS,
                 "nsaction: ",
             ),
+            ExceptionMessage(
+                TransactionException.NONCE_MISMATCH_TOO_HIGH,
+                "saction: ",
+            ),
+            ExceptionMessage(
+                TransactionException.NONCE_MISMATCH_TOO_LOW,
+                "action: ",
```

Comment:
When are we having normal exception strings?

Reply:
Yes it's getting a bit ridiculous at this moment. It needs to be refactored on EELS side, I'll open an issue on there in a bit.